### ISSUE TITLE
REDIS error while registering on mypod.store

### ISSUE THREAD

**Issue by mrkvon** [activitypods/activitypods#288]
Hello! I tried to register to mypod.store today, but received the following error:

> MISCONF Redis is configured to save RDB snapshots, but it's currently unable to persist to disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.

The error seems to be returned by `POST https://mypod.store/auth/signup` with response status `500` and body

```json
{"name":"ReplyError","message":"MISCONF Redis is configured to save RDB snapshots, but it's currently unable to persist to disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.","code":500}
```

This also seems to partially register the account: I can't sign in with the new account, yet I can't sign up again with the same username or email.

**Comment by Laurin-W** [comment]
Hey, nice to meet you and thanks for raising the issue!

Turns out, the disk was full...
I freed up some space for now and had to do a restart. However, the account you created is probably stuck in an inbetween state which might render it unusable. You can write me your username here or on matrix and I can remove it from the db manually. Alternatively, you can create a new account with a different username and email.

Sorry for the inconvenience!
Best, Laurin

**Comment by mrkvon** [comment]
Hello @Laurin-W, thank you for the response and addressing the issue!

The username I used was the same as here (email m...........a@p..........c..) if you care to remove it. Otherwise I'll try with another one.

FYI I just tried to register with different credentials, but the `POST https://mypod.store/auth/signup` seems to be returning either `502 Bad Gateway` or `404` with body

```json
{"name":"ServiceNotFoundError","message":"Service 'auth.authenticate' is not found.","code":404,"type":"SERVICE_NOT_FOUND","data":{"action":"auth.authenticate"}}
```

So signup still doesn't seem to be possible.
